common/supplemental/coverageLevels.xml

```diff
@@ -180,24 +180,42 @@ For terms of use, see http://www.unicode.org/copyright.html
 
 		<coverageVariable key="%anyAttribute" value='([^\x{22}]++)'/>
 
-		<!-- Number system coverage by language; default number system items at various levels, native system items all at modern -->
-		<coverageVariable key="%arabDefaultLanguages" value="(ar|ckb)"/>
-		<coverageVariable key="%arabextDefaultLanguages" value="(fa|ks|lrc|pa|ps|ur|uz)"/>
-		<coverageVariable key="%arabextNativeLanguages" value="(mzn|ug)"/>
-		<coverageVariable key="%bengDefaultLanguages" value="(as|bn)"/>
-		<coverageVariable key="%devaDefaultLanguages" value="(mr|ne)"/>
-		<coverageVariable key="%devaNativeLanguages" value="(brx|hi|kok)"/>
-		<coverageVariable key="%gujrNativeLanguages" value="(gu)"/>
-		<coverageVariable key="%guruNativeLanguages" value="(pa)"/>
-		<coverageVariable key="%hanidecNativeLanguages" value="(yue|zh)"/>
+		<!-- Number system coverage by language and/or script; default number system items at various levels, native & finance system items all at modern -->
+		<coverageVariable key="%adlmDefaultScripts" value="(Adlm)"/>
```

Comment:
I still don't think any of this belongs in data at all.  It should be read from the locale files themselves and cached.  But.. it's an improvement.

Reply:
Covered by https://unicode-org.atlassian.net/browse/CLDR-17618